exif/exif.go

```diff
@@ -27,6 +27,8 @@ const (
 	interopPointer = 0xA005
 )
 
+const APP_SEC_READ_BUFFER = 8 * 1024 // 8 KiB
```

Comment:
Why don't we hardcode 32K just as in the stdlib copybuffer? See https://golang.org/src/io/io.go#L387. IMHO this public constant should not be visible (or hardcoded)